external-builds/pytorch/skipped_tests.py

```diff
@@ -10,6 +10,8 @@
     "test_host_memory_stats",
     "test_nvtx",
     "test_device_count_not_cached_pre_init",
+    # TestCuda under test_cuda.py, failing on gfx942 (#1143)
+    "test_float32_matmul_precision_get_set ",
```

Comment:
If you have the logs from this test failing handy, could you paste a snippet from them into the PR description or at least this comment thread? It will take more effort to collect such logs later, once we lose the trail and logs start to expire.

Reply:
Ah, logs are on https://github.com/ROCm/TheRock/issues/1143 and https://github.com/ROCm/TheRock/actions/runs/16575535903/job/46878781147. Thanks.